packages/plugin-meta-extractor/README.md

```diff
@@ -0,0 +1,92 @@
+# Grafana / Plugin Meta Extractor
+
+`@grafana/plugin-meta-extractor` is a small package that can be used to extract meta-information from the source code of a Grafana plugin. The main use case is to be able to automatically generate static information for a plugin without the need for manual intervention.
```

Comment:
I think what is missing in the readme is:
- why this is needed. Yes we say "generate static information" but probably need more explanation to why
- How often and when this should run and where (locally, ci, etc.)

Reply:
I updated the intro a bit in https://github.com/grafana/plugin-tools/pull/805/commits/4162173898580b4ce30f73c6bc3ce261cb8d08a4 - is this something similar to what you were thinking about?